src/fides/core/utils.py

```diff
@@ -58,8 +59,15 @@ def validate_db_engine(connection_string: str) -> None:
     """
     Use SQLAlchemy to create a DB engine.
     """
+    # Pymssql doesn't support this arg
+    connect_args: Dict[str, Any] = (
+        {"connect_timeout": 10} if "pymssql" not in connection_string else {}
+    )
+    if "redshift" in connection_string:
+        connect_args["sslmode"] = "prefer"
+        connect_args["connect_timeout"] = 60
```

Comment:
The Redshift datasets from Axios were taking forever to load, this number let it run long enough to respond with all of the dataset information